### ISSUE TITLE
Css as string

### ISSUE THREAD

**Issue by Diokuz** [addyosmani/critical#291]
I have css as string, no file. Is there a way to tell `critical` to get css string, not the file?

```
const css = 'div { color: red }'
const html = '<div>Hello</div>'

critical({ html, css }, function(err, output) {})
```

**Comment by bezoerb** [comment]
https://github.com/addyosmani/critical/issues/584

**Comment by bezoerb** [comment]
Should be fixed in [v7.1.0](https://github.com/addyosmani/critical/releases/tag/v7.1.0)
You should now be able to also pass plain css to critical using the css option:
```
generate({
  css: `html{display:block;}.someclass{color:red}`
}, (err, output) => {
    console.log(output)
});
```
or via CLI:
```
critical http://my-domain.com -c 'html{display:block;}.someclass{color:red}'
```

Closing this now. 
if there are any issues with the new feature please continue in #588
